yasa/plotting.py

```diff
@@ -406,13 +402,13 @@ def topoplot(
     plt.rcParams.update({"savefig.transparent": "True"})
 
     # Make sure we don't do any in-place modification
-    assert isinstance(data, pd.Series), "Data must be a Pandas Series"
+    assert isinstance(data, pd.Series), "`data` must be a Pandas Series"
```

Comment:
Thanks for doing that!

Reply:
Yup they confused me at first but once I saw one in practice I liked it. Helps to identify what's a variable and what's not. 💯